src/redisearch_rs/c_entrypoint/iterators_ffi/src/wrapper.rs

```diff
@@ -7,213 +7,205 @@
  * GNU Affero General Public License v3 (AGPLv3).
 */
 
- use ffi::{
-     IteratorStatus, IteratorStatus_ITERATOR_EOF, IteratorStatus_ITERATOR_NOTFOUND,
-     IteratorStatus_ITERATOR_OK, IteratorStatus_ITERATOR_TIMEOUT, QueryIterator, ValidateStatus,
-     ValidateStatus_VALIDATE_ABORTED, ValidateStatus_VALIDATE_MOVED, ValidateStatus_VALIDATE_OK,
-     t_docId,
- };
- use inverted_index::RSIndexResult;
- use rqe_iterators::{RQEIterator, RQEIteratorError, RQEValidateStatus, SkipToOutcome};
- use std::ptr;
+use ffi::{
```

Comment:
This file is going away, see https://github.com/RediSearch/RediSearch/pull/7599 so I suppose those changes should now be in `src/redisearch_rs/rqe_iterators_interop/src/lib.rs`